docs/ops.html

```diff
@@ -3967,15 +3964,6 @@ <h4 class="anchor-heading"><a id="kraft_deployment" class="anchor-link"></a><a h
     <li>The Kafka controllers store all the metadata for the cluster in memory and on disk. We believe that for a typical Kafka cluster 5GB of main memory and 5GB of disk space on the metadata log director is sufficient.</li>
   </ul>
 
-  <h4 class="anchor-heading"><a id="kraft_missing" class="anchor-link"></a><a href="#kraft_missing">Missing Features</a></h4>
```

Comment:
Could you please remove `kraft missing` link from `toc.html` as well?

https://github.com/apache/kafka/blob/3.9/docs/toc.html#L178